views/fe-utils/countUtil.js

```diff
@@ -4,8 +4,20 @@ const getCounts = ({ acl }) => {
     options.push(...acl[study].subjects);
   }
   return {
-    totalStudies: acl.length,
-    totalSubjects: options.length,
+    // study determination: at least one upper case
```

Comment:
The logic is the same for both, what do you think about making this a function export function in the countUtil.js file

Reply:
I agree--codes are repeated in two files since DPdash was born. I do not have the bandwidth now to put them in one place. If you know how to do it, feel free to submit a pull request.

Reply:
I agree that we should be consolidating this logic in one place so it only has to be changed in one place next time.